lib/spotify_sdk_web.dart

```diff
@@ -195,9 +191,23 @@ class SpotifySdkPlugin {
           return false;
         }
       case MethodNames.getAuthenticationToken:
+        var clientId = call.arguments[ParamNames.clientId] as String?;
+        var redirectUrl = call.arguments[ParamNames.redirectUrl] as String?;
+
+        // ensure that required arguments are present
+        if (!(clientId?.isNotEmpty == true &&
+            redirectUrl?.isNotEmpty == true)) {
```

Comment:
```suggestion
        if (clientId == null ||
            clientId.isEmpty ||
            redirectUrl == null ||
            redirectUrl.isEmpty) {
```


Better do it this way. This will allow you to remove the forced unwrapping down below.